src/AppBundle/Controller/Utils/AdminDashboardTrait.php

```diff
@@ -27,6 +29,10 @@
 
 trait AdminDashboardTrait
 {
+
+    /** @var SlugifyInterface */
+    protected $slugify;
+
     protected function redirectToDashboard(Request $request)
```

Comment:
Thanks to dependency injection, you can just add a `SlugifyInterface $slugify` parameter to this method, and it will automagically be injected 😉